contxt/services/ems.py

```diff
@@ -29,6 +37,29 @@ def __init__(self, auth: Auth, env: str = "production", **kwargs) -> None:
     def get_facility(self, id: int) -> Facility:
         return Facility.from_api(self.get(f"facilities/{id}"))
 
+    def get_metric_values(
+        self,
+        asset_id: str,
+        metric_label: Optional[str] = None,
+        params: Optional[dict] = None,
+        page_options: Optional[PageOptions] = None,
+    ) -> Iterable[MetricValue]:
+        assert (
+            params is None
+            or sum([key not in ["effective_end_date", "effective_start_date"] for key in params]) == 0
+        ), f"Unrecognized query parameters: {params}"
+        params_string = (
+            "&".join([f"{key}={value}" for key, value in params.items()]) if params is not None else ""
+        )
+
+        url = (
+            f"assets/metrics/values?asset_ids={asset_id}"
+            + f"&metric_labels={metric_label}&{params_string}"
+        )
+        resp = self.get(url)
```

Comment:
`requests` will do this formatting for us with [`params`](https://docs.python-requests.org/en/latest/user/quickstart/#passing-parameters-in-urls):
```suggestion
        resp = self.get(
            "assets/metrics/values",
            params={"asset_ids": asset_id, "metric_labels": metric_label, **(params or {})},
        )
```